birdhouse/env.local.example

```diff
@@ -237,6 +237,9 @@ export GEOSERVER_ADMIN_PASSWORD=geoserverpass
 # Name of the images displayed on the JupyterHub image selection page
 # The name order must correspond to the order of the DOCKER_NOTEBOOK_IMAGES variable,
 # and both variables should have the same number of entries.
+# Note that the selection names are also used as directory names for the tutorial-notebooks directories mounted when
+# starting the corresponding image. The name can use the '<name>' or the '<name>:<version>' format. The version will be
+# excluded when mounting the corresponding directory.
 #export JUPYTERHUB_IMAGE_SELECTION_NAMES="pavics \
 #                                         eo-crim \
 #                                         nlp-crim"
```

Comment:
So basically here we can write `eo-crim` or `eo-crim:0.3.0` ?  If yes, then maybe juste provide directly as example so it is clear to the user?  Some users just do not read the comments :(

Reply:
Yes, both formats are indeed supported. 
I added an update with an example of selection name that uses the format with the version, as suggested. :)